dockers/broad/samtools_picard_bwa/README.md

```diff
@@ -0,0 +1,48 @@
+# Samtools_BWA_Picard
+
+## Quick reference
+
+Copy and paste to pull this image
+
+#### `docker pull us.gcr.io/broad-gotc-prod/samtools-picard-bwa:1.0.0-0.7.15-2.23.8-1626449438`
+`
+
+- __What is this image:__ This image is a lightweight alpine-based custom image for running Samtools, BWA and Picard, it uses `us.gcr.io/broad-gotc-prod/samtools` as a base image.
+- __What are Samtools, BWA and Picard:__ BWA is a software package for mapping DNA sequewnces against a large reference genome, [more info](https://github.com/lh3/bwa). Picard is a set of command line tools for manipulating high-throughput sequencing (HTS) data and formats, [more info](https://github.com/broadinstitute/picard). See Samtools README for [more info](../samtools/README.md).
+- __How to see tool version used in image:__ Please see below.
+
+## Versioning
+
+Samtools_BWA_Picard
```

Comment:
for consistency, should this line be `The samtools_picard_bwa image uses the following convention for versioning:`?